Rakefile

```diff
@@ -0,0 +1,12 @@
+require 'data_mapper'
+require './app/data_mapper_setup'
+
+task :auto_upgrade do
+  DataMapper.auto_upgrade!
+  puts "Auto-upgrade complete (no data loss)"
+end
+
+task :auto_migrate do
+  DataMapper.auto_migrate!
+  puts "Auto-migrate complete (data could have been lost)"
+end
```

Comment:
Now the migration will work. It doesn't work for the test database though, so you can do it like this: rake auto_migrate RACK_ENV=test.